### PR TITLE
Set admob banner offset to 0

### PR DESCRIPTION
**Pull request by devgiordane**
@4ian as mentioned in this PR.
https://github.com/4ian/GDevelop/pull/6890

### CODE REVIEW

**Comment by ClementPasteau** [comment]
thanks @devgiordane !

From reading the other PR you linked, you can probably remove the parameter from the action, as it's not used.
We'll introduce it later if needed

File: Extensions/AdMob/JsExtension.js [hunk 1]
```diff
@@ -302,6 +302,15 @@ module.exports = {
...
         false
       )
+      .addParameter(
+        'yesorno',
+        _('Offset? (outside webview otherwise)'),
+        '',
+        false
+      )
+      .setParameterLongDescription(
+        'By setting offset to true, just overlapping the banner on top of the content.\nkeep false for the default setting where the banner pushes the webview, this setting creates a black space around the banner, and can also create borders around your game.'
```
Comment (AlexandreSi) [inline Extensions/AdMob/JsExtension.js:312]: This sentence should be translated. You can do so by using `_(...)`  as done for the parameter label just above

File: Extensions/AdMob/admobtools.ts [hunk 2]
```diff
@@ -333,6 +333,7 @@ namespace gdjs {
         adUnitId,
         position: atTop ? 'top' : 'bottom',
         size: bannerRequestedAdSizeType,
+        offset: 0,
```
Comment (AlexandreSi) [inline Extensions/AdMob/admobtools.ts:336]: You are not using the parameter you added in the action declaration. It should be added in the `setupBanner` signature

Comment (4ian) [reply Extensions/AdMob/admobtools.ts:336]: As discussed on the other PR, this is valid but the parameter can be removed entirely ;)

File: Extensions/AdMob/JsExtension.js (hunk 1 above)
Comment (4ian) [reply Extensions/AdMob/JsExtension.js:312]: @devgiordane Can you remove this entirely?

Comment (devgiordane) [reply Extensions/AdMob/JsExtension.js:312]: Hello, I tried to remove this change, and keep only the offset commit, I have little familiarity with git, and I don't know if it worked.

Comment (4ian) [reply Extensions/AdMob/JsExtension.js]: It did not but I used GitHub to directly edit the file and remove the changes :)

**Comment by 4ian** [comment]
Merged! Thank you very much :)

**Pull request merged**